public/app/brothers/brothers.js

```diff
@@ -0,0 +1,8 @@
+angular.module('KDRPoints')
+.controller('BrothersCtrl', ['$scope', '$http', function($scope, $http) {
+  $http.get('/brothers')
```

Comment:
also relevant- brothers should be resolved (in app.module.js, like user on the root state), instead of pulled here